lib/bot.js

```diff
@@ -42,14 +42,14 @@ function getBaseUrl(prjName, config) {
   return configUrl;
 }
 
-
-function generateMsg(configUrl, issue) {
-  return ':jira: <' + configUrl + issue.key + '|*' + issue.key + '*>';
+function generateMsg(config, prjName, issue) {
```

Comment:
Why do we need to generate the baseUrl again?  How about just passing in the emoji and the configUrl?